hera_sim/tests/test_polybeam.py

```diff
@@ -179,3 +179,8 @@ def test_perturbed_polybeam(self):
         # Check that attempting to use GPU with MPI raises an error.
         with pytest.raises(RuntimeError):
             run_sim(r, use_gpu=True, use_mpi=True)
+
+
+def test_polybeam():
```

Comment:
try to make the test name a bit more informative: maybe `test_polybeam_kwargs`

Reply:
Thanks @steven-murray . I have modified the function name.